Feisty/GameDetails/GameDetailsViewController.swift

```diff
@@ -49,6 +58,15 @@ class GameDetailsViewController: UIViewController {
 
   }
   
+  private func setUpGameCardDropShadow() {
+    
+    gameCard.layer.shadowColor = UIColor.black.cgColor
```

Comment:
Shouldn't you use the extension you created here?

Reply:
Same issue as before with it being created in a later pull request. Will haver to go back and change it once it is in development